kustomization.yml

```diff
@@ -0,0 +1,14 @@
+kind: Kustomization
```

Comment:
You should make this optional. Installation will fail if kustomize.config.k8s.io/v1beta1 does not exist